### ISSUE TITLE
Dedicated packageType for packages with ./tools folder

### ISSUE THREAD

**Issue by matkoch** [NuGet/Home#10833]
## Feature Request

A lot of packages are purely used for what they ship in the `./tools` folder (related feature here is `PackageDownload`). It would be nice to have the ability to distinguish such packages on the metadata level (`packageType`).

Related to this request is #10782.

**Comment by joelverhagen** [comment]
One thing we need to be careful about is packages that contain both `tools` and `lib`. In such cases the package author might intend to support both the `PackageReference` and `PackageDownload` scenario with the same package.

Looking at the data, about 45% of packages with non-.ps1/.psm1/_._ tools files also have lib files.
![image](https://user-images.githubusercontent.com/94054/118303260-1a0db000-b49a-11eb-829b-1188775eeaa0.png)

So it seems to me that the chance of having a dual purpose package (i.e. both a Dependency package type and "tools" package type) is relatively high.  Due to this,  I think we may be blocked on this feature by https://github.com/NuGet/Home/issues/10468.
